app/controllers/projects/settings/general_controller.rb

```diff
@@ -46,4 +48,19 @@ def toggle_public
 
     redirect_to action: :show, status: :see_other
   end
+
+  def update
+    call = Projects::UpdateService
+      .new(model: @project, user: current_user)
+      .call(permitted_params.project)
+
+    @project = call.result
+
+    if call.success?
+      flash[:notice] = I18n.t(:notice_successful_update)
+      redirect_to project_settings_general_path(@project)
+    else
+      render action: :show, status: :unprocessable_entity
```

Comment:
@marcalcobe @psatyal 

Is it enough to display inline validation failures (as currently implemented), or should we also display a flash message? (e.g. by adding `flash[:error] = call.message`)

<img width="726" alt="image" src="https://github.com/user-attachments/assets/3d3e1f34-692c-44b8-b8a0-0a3cb10c16cc" />

<img width="1073" alt="image" src="https://github.com/user-attachments/assets/936cd39e-8ce9-4243-b21b-c54b50d15694" />

Reply:
It looks like `ErrorMessageHelper#error_messages_for` lets us accomplish the same thing.